### PR TITLE
Update examples to correct route capsules

### PR DESCRIPTION
**Pull request by tfpauly**
I realized that our example for the connection racing still had split route capsules. This updates the route advertisement capsule examples to show how they contain multiple "IP Address Range" structs

### CODE REVIEW

**Review by mirjak — APPROVED** [review]
Ups. Maybe a reason to resubmit on Monday to avoid confusion.

**Comment by tfpauly** [comment]
Yeah, it's not too critical, but we could

File: draft-age-masque-connect-ip.md [hunk 1]
```diff
@@ -640,17 +640,14 @@ Context Extension = {}
...
-                              STREAM(44): CAPSULE
-                              Capsule Type = ROUTE_ADVERTISEMENT
-                              IP Version = 6
-                              Start IP Address = 2001:db8::3456
-                              End IP Address = 2001:db8::3456
-                              IP Protocol = 17
+                              (IP Version = 4
+                               Start IP Address = 198.51.100.2
+                               End IP Address = 198.51.100.2
+                               IP Protocol = 17)
```
Comment (DavidSchinazi) [inline draft-age-masque-connect-ip.md:646]:
```suggestion
                               IP Protocol = 17),
```

**Review by DavidSchinazi — APPROVED** [review]
Approved - ideally with the comma but OK if you prefer without it

**Pull request merged**